server/database/schema.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE reviews (
   name VARCHAR(15) NOT NULL,
```

Comment:
@Matt-Wigg you probably want to define a primary id here. It'll help when you want to expand your db.

Reply:
@andrewdeanyoder Totally get what you are saying, but they are first name entries only - I don't want to limit uniqueness on a name basis (people could have the same first name but not be the same person).

Reply:
@Matt-Wigg right, I actually meant you might want to add a column that's the primary id. (I didn't know how to comment on this document without selecting a line somewhere. I didn't mean to select the first names. sorry for the confusion.)

Reply:
@andrewdeanyoder Here's my full Schema - 
  id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
  url_id INT NOT NULL,
  name VARCHAR(15) NOT NULL,
  location VARCHAR(50) NOT NULL,
  date DATETIME NOT NULL,
  comment VARCHAR(8000) NOT NULL,
  rating INT NOT NULL,
  helpful INT DEFAULT 0,
  img VARCHAR(225)